packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -223,10 +261,12 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
         ?.getLocalSecondary()
         ?.putValue(AT_PKAM_PUBLIC_KEY, atKeysMap[AuthKeyType.pkamPublicKey]!);
     logger.finer('PkamPublicKey persist to localSecondary: status $response');
-    response = await _atClient
-        ?.getLocalSecondary()
-        ?.putValue(AT_PKAM_PRIVATE_KEY, atKeysMap[AuthKeyType.pkamPrivateKey]!);
-    logger.finer('PkamPrivateKey persist to localSecondary: status $response');
+    if (atOnboardingPreference.authMode == PkamAuthMode.keysFile) {
```

Comment:
Need an explanation somewhere of why we are doing these special things when the authMode is PkamAuthMode.keysFile

Reply:
done